grafast/grafserv/src/servers/whtatwg-node-server/index.ts

```diff
@@ -0,0 +1,115 @@
+import { createServerAdapter } from '@whatwg-node/server'
+
+import { GrafservBase } from "../../core/base.js";
+import type {
+  GrafservConfig,
+  RequestDigest,
+  Result,
+} from "../../interfaces.js";
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Grafast {
+    interface RequestContext {
+        whatwg: {
+            request: Request
+        }
+    }
+  }
+}
+
+/** @experimental */
+export class WhatwgGrafserv extends GrafservBase {
+  protected whatwgRequestToGrafserv(
+    request: Request
+  ): RequestDigest {
+    const url = new URL(request.url);
+    return {
+      httpVersionMajor: 1,
+      httpVersionMinor: 0,
+      isSecure: url.protocol === 'https://',
```

Comment:
Are you sure this is correct? The protocol is actually `https:`, the `//` is not part of the protocol itself (see for example `location.protocol` in a browser).

https://developer.mozilla.org/en-US/docs/Web/API/URL/protocol

Reply:
Correct, fixed.